dashboard/src/gql/app/prefetchNewApp.graphql

```diff
@@ -32,7 +32,7 @@ query PrefetchNewApp {
   regions(order_by: { city: asc }) {
     ...PrefetchNewAppRegions
   }
-  plans(order_by: { sort: asc }) {
+  plans(order_by: {sort: asc}, where: {deprecated: {_eq: false}}) {
     ...PrefetchNewAppPlans
```

Comment:
I think there's a formatting issue here as there should be a space after `{`

Reply:
I am not sure I follow, it follows the same format as with `{sort: asc}`, for instance. Do we have a linter?